integration-templates/instantly/actions/set-campaign-name.ts

```diff
@@ -0,0 +1,45 @@
+import type { NangoAction, InstantlySetCampaignNameResponse, InstantlySetCampaignNameInput } from '../../models';
+
+export default async function runAction(nango: NangoAction, input: InstantlySetCampaignNameInput): Promise<InstantlySetCampaignNameResponse> {
+    if (!input.campaign_id) {
+        throw new nango.ActionError({
+            message: 'campaign_id is a required field'
+        });
+    } else if (!input.name) {
+        throw new nango.ActionError({
+            message: 'name is a required field'
+        });
+    }
+
+    try {
+        const connection = await nango.getConnection();
+
+        let api_key: string;
+        if ('apiKey' in connection.credentials) {
+            api_key = connection.credentials.apiKey;
+        } else {
+            throw new nango.ActionError({
+                message: `API key credentials is incomplete`
+            });
+        }
```

Comment:
nitpicking: we can probably use the null coaleasce operator here:
```
const api_key: string = connection.credentials.apiKey ?? 
  throw new nango.ActionError({
    message: 'API key credentials is incomplete'
  });
```

Reply:
@TBonnin TypeScript's strict type checking wouldn't be happy here. An error is raised since ts can't guarantee that `credentials` is of type `ApiKeyCredentials`